about.html

```diff
@@ -147,7 +147,7 @@ <h2 class="heading text-center">Partners</h2>
             </div>
         </div>
     </footer>
-    <div class="copy container text-black">
+    <div class="copy  text-black about-footer">
             <div class="logo">
                  <a href="index.html">Hooge Media</a>
             </div>
```

Comment:
# Optional: 
- Please move the text to the right side to avoid it staying in the middle
- 
![2](https://user-images.githubusercontent.com/63972670/197906144-02bc49c5-4948-461c-9e56-54e1081fe632.JPG)